tests/system/conftest.py

```diff
@@ -51,7 +51,14 @@ def pytest_collection_modifyitems(config: pytest.Config, items: list[pytest.Item
     rootdir = config.rootpath
     for item in items:
         rel_path = item.path.relative_to(rootdir)
+
+        # Provider system tests
         match = re.match(".+/system/providers/([^/]+)", str(rel_path))
         if match:
             provider = match.group(1)
             item.add_marker(pytest.mark.system(provider))
+
+        # Core system tests
+        match = re.match(".+/system/[^/]+", str(rel_path))
+        if match:
+            item.add_marker(pytest.mark.system("core"))
```

Comment:
Why do we need this? Couldn't we just put the system test in the `common` provider and it would be picked up as normal? Or are we worried that actual system tests would eventually live there and get accidentally run by the Airlfow CI?

Reply:
As far as I understand, `common` is not a provider in itself. `common-io` and common-sql` are. So putting the system test in either of the two does not really makes sense to me. That's why I preferred putting it in core Airflow.

Reply:
Got it, thanks!